### DISCUSSION TITLE
Additional data for timeline tweets?

### DISCUSSION THREAD

**Discussion by jormungander** [redouane59/twittered#339] [Q&A] [answered]
The user timeline request does not have the MEDIA_FIELD and EXPANSION parameters set, unlike many of the other tweet requests.  Is this done deliberately?

**Comment by redouane59** [comment, marked as answer]
Hello @jormungander , i'm going to check now. I'll keep you informed.

**Reply by redouane59** [reply]
Missing fields added :) 
https://github.com/redouane59/twittered/pull/341 
Will be available in the next release

**Reply by jormungander** [reply]
That's great, thanks!
